src/cli.ts

```diff
@@ -42,30 +44,39 @@ function isValidUrl(proposedUrl) {
   }
 }
 
-function exitWithError(errorMessage) {
+function exitWithError(errorMessage: string) {
   console.error(errorMessage);
   process.exitCode = 1;
 }
 
-async function readChangelog(changelogPath) {
+async function readChangelog(changelogPath: string) {
   return await fs.readFile(changelogPath, {
     encoding: 'utf8',
   });
 }
 
-async function saveChangelog(changelogPath, newChangelogContent) {
+async function saveChangelog(changelogPath: string, newChangelogContent = '') {
   await fs.writeFile(changelogPath, newChangelogContent);
 }
 
+interface UpdateOptions {
+  changelogPath: string;
+  currentVersion?: Version;
+  repoUrl: string;
+  isReleaseCandidate: boolean;
+  projectRootDirectory?: string;
+}
+
 async function update({
   changelogPath,
   currentVersion,
   isReleaseCandidate,
   repoUrl,
   projectRootDirectory,
-}) {
+}: UpdateOptions) {
   const changelogContent = await readChangelog(changelogPath);
 
+  // TODO: This can be undefined. Is that OK?
```

Comment:
`updateChangelog` can return `undefined`. Should we save that content?

Reply:
Ah, good catch! This was a mistake. We should not save anything in this case.

Reply:
Fixed in 47e9a7a